### PR TITLE
W1d1

### PR DESCRIPTION
**Pull request by pdmxdd**
Walkthrough: Unit Tests
Walkthrough: GitLab
Configuration: IntelliJ

### CODE REVIEW

**Comment by pdmxdd** [comment]
Studio: Junit

File: src/walkthroughs/unit-tests/index.rst [hunk 1]
```diff
@@ -7,28 +7,121 @@ Walkthrough: Unit Tests
...
+* Open IntelliJ
+* Choose **open folder** and navigate to and choose folder **car-unit-test-starter**
+
+
+Configure IntelliJ for Java 8
+-----------------------------
+
+Before we can write any unit tests we need to configure this project for Java 8.
+
+Check out the `Configure IntelliJ <../../configurations/intellij/>`_ article for detailed steps.
```
Comment (welzie) [inline src/walkthroughs/unit-tests/index.rst:35]: This link didn't work for me.

File: src/walkthroughs/unit-tests/index.rst [hunk 2]
```diff
@@ -7,28 +7,121 @@ Walkthrough: Unit Tests
...
+
+Write and Run our Unit Tests
+----------------------------
+* Write tests
+
+.. image:: /_static/images/unit-tests/completed-tests.png
+
+Our first three tests are very similar to one another. They each have a test_car object, and are checking that the gas tank level is correct after instantiating the car, driving the car within the range, and driving the car exceeding the range.
+
+The assertEquals() method takes three parameters: The value we want to test, What it's value should be, and the delta (how much the actual, and expected values can differ and still return true).
```
Comment (welzie) [inline src/walkthroughs/unit-tests/index.rst]: IMHO functions and code like **assertEquals()** should be inside an inline code style  ``assertEquals()`` by wrapping it with double ` on each side

File: src/week01/day1/index.rst [hunk 3]
```diff
@@ -45,6 +45,8 @@ Exercises
 Studio
 ------
 
+* :ref:`junit-studio`
+
 Studio Setup
```
Comment (welzie) [inline src/week01/day1/index.rst]: Should the **Studio Setup**  and the **Turning in the Studio** be moved into the Studio itself?

**Review by welzie — COMMENTED** [review]
I left a few comments below. I have also fixed a few items that you can see in this PR https://github.com/LaunchCodeEducation/gis-devops/pull/13

File: src/walkthroughs/unit-tests/index.rst (hunk 1 above)
Comment (pdmxdd) [reply src/walkthroughs/unit-tests/index.rst:35]: Configuration has been added.

File: src/walkthroughs/unit-tests/index.rst (hunk 2 above)
Comment (pdmxdd) [reply src/walkthroughs/unit-tests/index.rst]: Methods have been wrapped by double 1.

**Comment by pdmxdd** [comment]
I added configurations/intellij. Can you verify the file exists, is linked to properly, and review it's contents?

**Review by welzie — APPROVED** [review]

**Pull request merged**